BMM.Core/Implementations/Player/ViewModelHandlingMediaPlayerDecorator.cs

```diff
@@ -87,13 +91,15 @@ public async Task Play(IList<IMediaTrack> mediaFiles, IMediaTrack currentTrack,
         {
             if (!ShowErrorIfOutdatedLiveRadio(currentTrack))
             {
+                _playbackHistoryService.AddPlayedTrack(currentTrack);
```

Comment:
what's the performance of this action? accessing the blobcache easily takes 100ms because of Json deserialization. But playing the song is a very time-sensitive action.

Reply:
It depends on device, I tested on my samsung and it takes less than 100ms with maximum 100 entries added. I haven't noticed any negative impact in UI.